src/containers/Login.vue

```diff
@@ -322,10 +282,10 @@ export default Vue.extend({
   max-width: 100%;
 }
 .top-bar {
-  padding: 59px 0px 0px 104px;
+  padding: 29px 0px 0px 104px;
```

Comment:
I know you didn't create these padding numbers, but could you update them to use rem values instead of pixel values throughout this file? And just as an FYI you might need to round them before converting to get nicer rem numbers (so maybe 28px -> 1.75rem).

Good article on why you should use rem if you're not familiar with it: https://engageinteractive.co.uk/blog/em-vs-rem-vs-px